cmd/kubeadm/app/util/version_test.go

```diff
@@ -269,9 +264,7 @@ func TestCIBuildVersion(t *testing.T) {
 		{"release/0invalid", "", false},
 		// CI or custom builds
 		{"ci/v1.9.0-alpha.1.123+acbcbfd53bfa0a", "v1.9.0-alpha.1.123+acbcbfd53bfa0a", true},
-		{"ci-cross/v1.9.0-alpha.1.123+acbcbfd53bfa0a", "v1.9.0-alpha.1.123+acbcbfd53bfa0a", true},
 		{"ci/1.9.0-alpha.1.123+acbcbfd53bfa0a", "v1.9.0-alpha.1.123+acbcbfd53bfa0a", true},
-		{"ci-cross/1.9.0-alpha.1.123+acbcbfd53bfa0a", "v1.9.0-alpha.1.123+acbcbfd53bfa0a", true},
 		{"ci/0invalid", "", false},
```

Comment:
can we have `k8s-master` here?

Reply:
This section tests release versions. However, afaiu k8s-master doesn't contain a specific version, but it keeps changing based on the latest cross builds available. So, it won't be possible to test k8s-master with a specific version. I guess that's the same reason why we don't test ci/latest in this particular section. Please correct me if my understanding is wrong here.

Reply:
SGTM.